### PR TITLE
update the docs about limiting compaction

### PR DESCRIPTION
**Pull request by annastuchlik**

### CODE REVIEW

**Comment by annastuchlik** [comment]
@raphaelsc Can you review it?

File: docs/source/use-monitoring/advisor/heavyCompaction.rst [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+If this is the case, you can do one of the following:
 
-1. Change ``scylla.yml`` to have ``compaction_static_shares: 100`` 
+* Statically limit the compaction shares with the ``compaction_static_shares`` option by setting a value between 50 and 1000:
 
-or
+    * In the ``scylla.yml`` configuration file: ``compaction_static_shares: 100``
+    * In the command line when starting ScyllDB: ``--compaction-static-shares 100``
+* Enforce ``min_threshold`` by setting ``compaction_enforce_min_threshold: true`` in the ``scylla.yml`` configuration file.
+  As a result, ScyllaDB will compact only the buckets that contain the number of SSTables specified with ``min_treshold``
```
Comment (raphaelsc) [inline docs/source/use-monitoring/advisor/heavyCompaction.rst]: typo. please replace it with min_threshold

**Comment by annastuchlik** [comment]
@raphaelsc Thanks. Fixed.

File: docs/source/use-monitoring/advisor/heavyCompaction.rst [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
-In those cases the compaction would impact the overall system performance.
+ScyllaDB runs compaction periodically as a background process. While running compaction is important, there are situations when
+compaction takes too much CPU.
+As a result, compaction impacts the overall system performance.
 
-When facing this problem, you can statically limit the compaction shares with one of two options:
+If this is the case, you can do one of the following:
 
-1. Change ``scylla.yml`` to have ``compaction_static_shares: 100`` 
+* Statically limit the compaction shares with the ``compaction_static_shares`` option by setting a value between 50 and 1000:
```
Comment (raphaelsc) [inline docs/source/use-monitoring/advisor/heavyCompaction.rst:10]: In addition to saying that the user can set optionally a value between 50 and 1000. we can say that 100 may be a good value to start with. additionally say that if read latency is impacted, it means that compaction is running too slow, then the user will have to increase compaction_static_shares until a sweet spot is reached

File: docs/source/use-monitoring/advisor/heavyCompaction.rst [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
 
-or
+    * In the ``scylla.yml`` configuration file: ``compaction_static_shares: 100``
+    * In the command line when starting ScyllDB: ``--compaction-static-shares 100``
```
Comment (raphaelsc) [inline docs/source/use-monitoring/advisor/heavyCompaction.rst:12]: another typo: ScyllDB -> ScyllaDB

other than that LGTM

**Comment by annastuchlik** [comment]
@raphaelsc I don't have permissions to reopen this PR, so I opened a new one: https://github.com/scylladb/scylla-monitoring/pull/1700

**Pull request merged**